tests/utils.go

```diff
@@ -1569,12 +1569,14 @@ func RunVMIAndExpectLaunch(vmi *v1.VirtualMachineInstance, timeout int) *v1.Virt
 }
 
 func RunVMIAndExpectLaunchWithDataVolume(vmi *v1.VirtualMachineInstance, dv *cdiv1.DataVolume, timeout int) *v1.VirtualMachineInstance {
-	obj := RunVMI(vmi, timeout)
+	_ = RunVMI(vmi, timeout)
 	By("Waiting until the DataVolume is ready")
 	Eventually(ThisDV(dv), timeout).Should(HaveSucceeded())
+	vmi, err := ThisVMI(vmi)()
+	Expect(err).To(BeNil())
 	By("Waiting until the VirtualMachineInstance will start")
-	WaitForSuccessfulVMIStartWithTimeout(obj, timeout)
-	return obj
+	WaitForSuccessfulVMIStartWithTimeout(vmi, timeout)
+	return vmi
```

Comment:
There's a warnings ignore list for this kind of scenario. 

```
type WarningsPolicy struct {
        FailOnWarnings     bool
        WarningsIgnoreList []string
}
```

The way this is written now will reset the warning watcher which could cause other warnings besides the one we actually care about ignoring to be ignored.